### PR TITLE
fix SortedCacheAddress to make work with JbossCacheClient

### PR DESCRIPTION
**Pull request by deedee**

### CODE REVIEW

File: src/java/main/com/topcoder/direct/services/view/util/SortedCacheAddress.java [hunk 1]
```diff
@@ -16,12 +16,14 @@
  *
  * @version 1.0
  */
-public class SortedCacheAddress implements CacheAddress {
+public class SortedCacheAddress implements JbossCacheAddress {
```
Comment (skyhit) [inline src/java/main/com/topcoder/direct/services/view/util/SortedCacheAddress.java]: @deedee this is incorrect, we should not tied to specific, like JBossCacheAddress, which seems specific to jboss cache, if we use Redis as the cache, will that cause problem?

**Comment by skyhit** [comment]
@deedee what is the purpose for this `SortedCacheAddress`? Basically, I think we should remove this class. or implement it inside the tc-cache.

File: src/java/main/com/topcoder/direct/services/view/util/SortedCacheAddress.java (hunk 1 above)
Comment (deedee) [reply src/java/main/com/topcoder/direct/services/view/util/SortedCacheAddress.java]: JbossCacheAddress extend CacheAddress. Implementing JbossCacheAddress  should work on both client. I think this approach make task more simple instead creating different classes for each clients and use AddressFactory to get it.

Comment (skyhit) [reply src/java/main/com/topcoder/direct/services/view/util/SortedCacheAddress.java]: @deedee ok

**Comment by deedee** [comment]
@skyhit fell free to move it. But I think this class was required for 2 reasons:
- To set exp time on jboss cache client, need to put inside JbossCacheAdress, where if we didn;t set this then max time will be used(current config set to 3days - too long).
- Better to have a  CacheAddress that can generate key from array of long, So we can use like gid. If not than key must be generate manually outside.

**Pull request merged**